core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -103,11 +108,12 @@ public void run() {
     try {
       createVersionMetric(metricsSystem);
       metricsEndpoint.start(vertx);
-
+      boolean failedToLoadSomeSigners = false;
       try {
         artifactSignerProvider.load().get(); // wait for signers to get loaded ...
       } catch (final InterruptedException | ExecutionException e) {
         LOG.error("Error loading signers", e);
+        failedToLoadSomeSigners = true;
```

Comment:
I don't think is going to catch all cases where signers aren't loaded. In the SignerLoader class we catch most the exceptions log errors and continue for the remaining signers.